README.md

```diff
@@ -64,19 +64,22 @@ The code is flexible and allows to condition a response by an arbitrary categori
 
 ## Quick start
 
-Quickly build a CPU-only docker image, run it & start the CakeChat serving the model on 8080 port:
+Run a CPU-only pre-built docker image & start the CakeChat serving the model on 8080 port:
 
 ```(bash)
-# Build & run docker container
-docker build -t cakechat:latest -f dockerfiles/Dockerfile.cpu dockerfiles/
+docker run --name cakechat-dev -p 127.0.0.1:8080:8080 -it lukalabs/cakechat:latest \
+    bash -c "python bin/cakechat_server.py"
+```
+
+(Or) with the GPU-enabled image:
 
-docker run --name cakechat-dev -p 127.0.0.1:8080:8080 -it cakechat:latest \
-    bash -c "python tools/download_model.py && python bin/cakechat_server.py"
+```(bash)
+nvidia-docker run --name cakechat-gpu-dev -p 127.0.0.1:8080:8080 -it lukalabs/cakechat-gpu:latest \
```

Comment:
Should be tested on few GPU-enabled configurations (maybe extra comments will be needed)

Reply:
Tested on several configurations:
— GPUs: Tesla K80, Tesla M60, GeForce GT 640
— NVIDIA Driver versions: 375.39, 384.111, 390.12
— CUDA 8.0 only